### ISSUE TITLE
Really Powerful Solar Panel

### ISSUE THREAD

**Issue by luke-a-childs** [Electrical-Age/ElectricalAge#850]
So i'm trying to set up a solar grid. I've got 9 rows of panels in a series connection using high voltage wires between them when I check the power output with my meter inside my house and realize it's now outputting 807 volts when before it was 398.

No wonder my battery exploded craziest part it's outputting that at night all the other panels are outputting a nominal 14.7 - 14.8 but this crazy panel is outputting 424 volts by it's self.

The problem persisted through closing and opening the world here is a screenshot of it outputting that value.
The problem also persisted through me removing the panel and placing it back.
![2018-05-15_09 21 44](https://user-images.githubusercontent.com/21255376/40062956-52b84ee4-582a-11e8-928e-59c518d40282.png)

**Comment by jrddunbr** [comment]
Can you place a ground pin on one side of it, and measure the voltage on the other side? It would be best if you separate the panel from the rest of the wiring to check. A version number of the mod would also be helpful. A larger screenshot showing more of your setup may be better as well. Perhaps something is backfeeding power.

**Comment by Gerson1813** [comment]
I do not know if I understand what you mean, but you did not specify how many panels you put in a series, show a larger image of your system, and add more details

**Comment by luke-a-childs** [comment]
Well the version is the latest stable release.

I can't send a screen shot as I destroyed that setup to make a new one that worked.

The setup was 9 panels in a row and three rows separated by one block and connected together by medium/high voltage.

All panels where connected only in series. The last two panels acted strangely the second the last one gave zero output the last one gave super high voltage.

Tho maybe it was a backfeed from a battery I'd placed only I reason I don't think it's that is that it happened after I placed it some time after I'd already placed multiple batteries down in my setup to get charged. (I'd broken and replaced the batteries a lot)

**Comment by Gerson1813** [comment]
9 serial panels were to be generating about 133V, not 800 your power may be going back through the transformer if you use and giving this high voltage

**Comment by Gerson1813** [comment]
9 serial panels were to be generating about 133V, not 800 your power may be going back through the transformer if you use and giving this high voltage

**Comment by Gerson1813** [comment]
maybe you did it make sure that you have properly connected the transformers
![2018-05-16_11 56 10](https://user-images.githubusercontent.com/29736511/40125060-54aefa32-5900-11e8-8441-2bfcd12cca94.png)

**Comment by luke-a-childs** [comment]
Remember it's three rows of nine so 27 panels in total and you should get about 398.

I was using no transformer at the time as I didn't finish with my setup because I noticed the insane voltage value.

To check if you're getting it right check the last two panels one should be outputting nothing and the last one about 412 volts

**Comment by Gerson1813** [comment]
27 panels in series? There are no batteries capable of withstanding more than 200 volts

**Comment by Gerson1813** [comment]
less that you use series batteries as well

**Comment by jrddunbr** [comment]
~~That's 3 rows of solar panels that have 9 panels. Not 27 panels in series.~~

Edit: derp

**Comment by jrddunbr** [comment]
9 panels in series generates 133v under no load. I think that may be a bit low for batteries.

**Comment by Gerson1813** [comment]
it seems to be in series 
![capturar](https://user-images.githubusercontent.com/29736511/40126161-065edbb0-5903-11e8-895c-dc33dbdcb7e5.PNG)

**Comment by luke-a-childs** [comment]
That was the plan I only had one battery in place at the time

**Comment by luke-a-childs** [comment]
Each panel is connected together.

The middle panel is flowing the opposite of the other two

**Comment by luke-a-childs** [comment]
So they are all in series

**Comment by jrddunbr** [comment]
I think that you connected the panels incorrectly initially. They are polarized. You probably connected the first 9 correctly, the second 9 backwards, and the 3rd group of 9 correctly again. The screenshot that you initially took backs up this claim - if you look closely at the texture, the panels on the right side look backwards.

**Comment by luke-a-childs** [comment]
The 2nd group is reversed so red flows to blue and blue to red again for the 3rd row

**Comment by jrddunbr** [comment]
Hmm. It seems like some anomaly to me that it output twice the voltage it should from all of the panels possible voltage either way. I think either the game made a derp (it does from time to time), or that something was backfeeding a voltage that was making everything blow up on you.

**Comment by luke-a-childs** [comment]
Nothing blew up since I was using high voltage thankfully neither did the battery that was down in the basement because I hadn't grounded the battery yet

**Comment by jrddunbr** [comment]
Hmm, I wonder if the battery did something weird, since it wasn't grounded. Although it shouldn't do anything in real life, we've been encountering oddities with ungrounded things recently.

**Comment by jrddunbr** [comment]
Also, are you aware of the 200v panels? They are nice for higher voltage solar.

**Comment by luke-a-childs** [comment]
Yes I'm aware I just didn't want to use them because of how massive they are

**Comment by luke-a-childs** [comment]
Hey where should I offer suggestions?

**Comment by jrddunbr** [comment]
People will see it if it's an issue on GitHub or in the support chat on Gitter or IRC (they are mirrors of one another). If there's already an issue about it, I would suggest tacking on to that.

**Comment by luke-a-childs** [comment]
Oh and about that "grounding" issue I grounded a high voltage wire and suddenly voltage, current, and power dropped to nothing and the wires exploded so?
